english/breakfast/breakfast.md

```diff
@@ -1,4 +1,5 @@
 You see a pot of coffee near you and you realize that you are hungry. Well, you poured coffee in a mug and picked some 
 marshmallows from the wall and added them to the coffee. While you are drinking the coffee and eating those delicious 
 marshmallows and staring at the wall you found that you made a whole in the wall by picking marshmallows. You finished
-your flavoured coffee and got out from marshmallows-walled room.
\ No newline at end of file
+your flavoured coffee and got out from marshmallows-walled room.
+Outside, a lush forest is echoing with birdsong and the cries of savage monkeys. The scent of marshmallows is fainter put still present and seems to be oozing from some place not far away.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.